Src/Eiffel/interface/new_graphical/text_window/text/autocomplete/eb_class_info_analyzer.e

```diff
@@ -435,6 +435,83 @@ feature {NONE}-- Clickable/Editable implementation
 			end
 		end
 
+	described_local (token: EDITOR_TOKEN; line: EDITOR_LINE; ft: FEATURE_AS; ft_start_token: detachable EDITOR_TOKEN_FEATURE_START): detachable TUPLE [feat: E_FEATURE; ast: detachable AST_EIFFEL]
+			-- search in feature represented by `ft' the feature associated with `token' if any
```

Comment:
Also the name `local' also covers arguments. In the compiler we talk about ACCESS_ID_AS which can be a local, an argument or a feature call. Might be good to merge the `describe_feature' and `describe_local' into one that does the check in the same order as the compiler (see AST_FEATURE_CHECKER_GENERATOR).